config/initializers/lograge.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Rails.application.configure do
+  config.lograge.enabled = true
+  config.lograge.formatter = Lograge::Formatters::Json.new
+  config.lograge.custom_options = lambda do |event|
+    {
+      request_id: event.payload[:headers]['action_dispatch.request_id']
```

Comment:
It seemed worth adding this one `custom_option` so that the `request_id` is within the JSON string and not outside.

Reply:
This is reasonable. It's also helpful for future projects that there's this demonstration of custom options. E.g., other apps may want to include user ids.